crates/biome_string_case/src/lib.rs

```diff
@@ -444,6 +630,12 @@ impl StrOnlyExtension for str {
         }
     }
 
+    fn ascii_nat_cmp(&self, other: &Self) -> Ordering {
```

Comment:
Docs missing

Reply:
The docs is inherited. OI don't think we should repeat it?

Reply:
My mistake :)